fhir/com.b2international.snowowl.fhir.core/src/org/linuxforhealth/fhir/model/r5/type/code/FHIRAllTypes.java

```diff
@@ -0,0 +1,4836 @@
+/*
+ * (C) Copyright IBM Corp. 2019, 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.linuxforhealth.fhir.model.r5.type.code;
+
+import org.linuxforhealth.fhir.model.annotation.System;
+import org.linuxforhealth.fhir.model.r5.type.Code;
+import org.linuxforhealth.fhir.model.r5.type.Extension;
+import org.linuxforhealth.fhir.model.r5.type.String;
+
+import java.util.Collection;
+import java.util.Objects;
+
+import javax.annotation.Generated;
+
+@System("http://hl7.org/fhir/fhir-types")
+@Generated("org.linuxforhealth.fhir.tools.CodeGenerator")
+public class FHIRAllTypes extends Code {
```

Comment:
Please remove the ones we do not support.

Reply:
OperationDefinition relies on this code system: https://github.com/b2ihealthcare/snow-owl/blob/89d4ab735d72832cbe7ced57175fc45f80ea894d/fhir/com.b2international.snowowl.fhir.core/src/org/linuxforhealth/fhir/model/r5/resource/OperationDefinition.java#L1980-L1993

Reply:
🆗